remove strip -s in makefile

Joerg Bornkessel <hd_brummy@gentoo.org> 06 April 2007
diff -Naur graphlcd-base-0.1.5.orig/tools/convpic/Makefile graphlcd-base-0.1.5/tools/convpic/Makefile
--- graphlcd-base-0.1.5.orig/tools/convpic/Makefile	2007-04-06 20:34:29.000000000 +0200
+++ graphlcd-base-0.1.5/tools/convpic/Makefile	2007-04-06 20:35:17.000000000 +0200
@@ -36,7 +36,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
diff -Naur graphlcd-base-0.1.5.orig/tools/crtfont/Makefile graphlcd-base-0.1.5/tools/crtfont/Makefile
--- graphlcd-base-0.1.5.orig/tools/crtfont/Makefile	2007-04-06 20:34:29.000000000 +0200
+++ graphlcd-base-0.1.5/tools/crtfont/Makefile	2007-04-06 20:35:32.000000000 +0200
@@ -35,7 +35,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
diff -Naur graphlcd-base-0.1.5.orig/tools/genfont/Makefile graphlcd-base-0.1.5/tools/genfont/Makefile
--- graphlcd-base-0.1.5.orig/tools/genfont/Makefile	2007-04-06 20:34:29.000000000 +0200
+++ graphlcd-base-0.1.5/tools/genfont/Makefile	2007-04-06 20:35:43.000000000 +0200
@@ -38,7 +38,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 	
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
diff -Naur graphlcd-base-0.1.5.orig/tools/lcdtestpattern/Makefile graphlcd-base-0.1.5/tools/lcdtestpattern/Makefile
--- graphlcd-base-0.1.5.orig/tools/lcdtestpattern/Makefile	2007-04-06 20:34:29.000000000 +0200
+++ graphlcd-base-0.1.5/tools/lcdtestpattern/Makefile	2007-04-06 20:35:57.000000000 +0200
@@ -36,7 +36,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
diff -Naur graphlcd-base-0.1.5.orig/tools/showpic/Makefile graphlcd-base-0.1.5/tools/showpic/Makefile
--- graphlcd-base-0.1.5.orig/tools/showpic/Makefile	2007-04-06 20:34:29.000000000 +0200
+++ graphlcd-base-0.1.5/tools/showpic/Makefile	2007-04-06 20:36:09.000000000 +0200
@@ -36,7 +36,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
diff -Naur graphlcd-base-0.1.5.orig/tools/showtext/Makefile graphlcd-base-0.1.5/tools/showtext/Makefile
--- graphlcd-base-0.1.5.orig/tools/showtext/Makefile	2007-04-06 20:34:29.000000000 +0200
+++ graphlcd-base-0.1.5/tools/showtext/Makefile	2007-04-06 20:36:21.000000000 +0200
@@ -36,7 +36,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
